scripts/kickstart_generator.rb

```diff
@@ -10,14 +10,15 @@ class KickstartGenerator
     KS_DIR     = "kickstarts"
     KS_GEN_DIR = "#{KS_DIR}/generated"
 
-    attr_reader :targets, :puddle, :git_checkout
+    attr_reader :targets, :puddle, :appliance_checkout, :manageiq_checkout
 
-    def initialize(build_base, targets, puddle, git_checkout)
-      @build_base   = Pathname.new(build_base)
-      @ks_gen_base  = @build_base.join(KS_GEN_DIR)
-      @targets      = targets
-      @puddle       = puddle # used during ERB evaluation
-      @git_checkout = git_checkout
+    def initialize(build_base, targets, puddle, manageiq_checkout, appliance_checkout)
```

Comment:
I don't know why, but the ordering bothers me that the attr_reader above and lines 20-21 below have appliance before manageiq, but these params are the other way around.